showcase-docs/getting-started.md

```diff
@@ -2,6 +2,152 @@
+
+If you don't have the `dynamic-plugins-rhdh` ConfigMap, you can create it with the following content:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: false
+```
+
+Don't forget to add `dynamicPluginsConfigMapName` referencing the above `ConfigMap` to your `Backstage` resource:
+
+```yaml
+spec:
+  application:
+    dynamicPluginsConfigMapName: dynamic-plugins-rhdh
+```
+
+### Customizing Telemetry Destination
+
+By default, the Segment plugin is configured to send data to Red Hat. If you wish to change the destination, you can do so by setting the `SEGMENT_WRITE_KEY` environment variable to the desired Segment write key.
+
+#### When using Helm Chart
+
+Add the following configuration to your
+
+```yaml
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: SEGMENT_WRITE_KEY
+        value: <segment_key>
+```
+
+#### When using RHDH Operator
+
+```yaml
+extraEnvs:
+  envs:
+    - name: SEGMENT_WRITE_KEY
+      value: <segment_key>
+```
+
+If you wish to subsequently disable telemetry data collection, use one of the following methods described below.
+
+### Disable Telemetry
+
+To turn off Segment telemetry, you need to deactivate the Segment provider plugin
+
+#### Using Helm Chart
+
+When you are using Helm Chart, you need to modify Helm values by adding the following configuration (if the Segment plugin is already present in your configuration, you can just change `disabled: false` to `disabled: true`):
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: true
+```
+
+#### When using RHDH Operator
+
+When using RHDH Operator, you need to modify the ConfigMaps used for dynamic plugin configuration. The name of this ConfigMap is specified in `dynamicPluginsConfigMapName` field in your `Backstage` CustomResource.
+Usually, it is named as `dynamic-plugins-rhdh`.
+To disable Segment provider plugin, add the following configuration (if the Segment plugin is already present in your configuration, you can just change `disabled: false` to `disabled: true`):
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: true
+```
+
```

Comment:
NOTE: If the `analytics-provider-segment` plugin is already present in your `ConfigMap` file, set the value of the `plugins.disabled` parameter to `true`.